src/test/java/com/cloudant/tests/ClientLoadTest.java

```diff
@@ -32,16 +32,16 @@
 	/** client max connections */
 	private static final int MAX_CONNECTIONS = 20;
 	
-	@BeforeClass 
-	public static void setUpClass() {
+	@Before
+	public  void setUpClass() {
```

Comment:
Do we need to do this before each test rather than once of the class? I'm unclear why we're now doing the database setup for most of the test classes `@Before` instead of `@BeforeClass`. If there's a reason for it being `@Before`, this method should probably be called  `setUp()`.